README.md

```diff
@@ -91,42 +91,27 @@ channels:
 
 ### TOC Liaisons
 
-- Saad Ali ([@saad-ali](https://github.com/saad-ali)), Google
-- Alena Prokharchyk ([@alena1108](https://github.com/alena1108)), Apple
-
-### Bootstrap TAG members
-
-- Amye Scavarda Perrin ([@amye](https://github.com/amye)), CNCF
-- April Kyle Nassi ([@thisisnotapril](https://github.com/thisisnotapril)), Google
-- Carolyn Van Slyck ([@carolynvs](https://github.com/carolynvs)), Microsoft
-- Cheryl Hung ([@oicheryl](https://github.com/oicheryl)), CNCF
-- Gerred Dillon ([@gerred](https://github.com/gerred)), Mycelial
-- Ihor Dvoretskyi ([@idvoretskyi](https://github.com/idvoretskyi)), CNCF
-- Josh Berkus ([@jberkus](https://github.com/jberkus)), Red Hat
-- Karen Chu ([@karenhchu](https://github.com/karenhchu)), Microsoft
-- Matt Klein ([@mattklein123](https://github.com/mattklein123)), Lyft
-- Paris Pittman ([@parispittman](https://github.com/parispittman)), Apple
-- Saad Ali ([@saad-ali](https://github.com/saad-ali)), Google
-- Stephen Augustus ([@justaugustus](https://github.com/justaugustus)), Cisco
-- Zach Corleissen ([@zacharysarah](https://github.com/zacharysarah)), Linux Foundation
+- Matt Farina ([@mattfarina](https://github.com/mattfarina)), SuSE
+- Katie Gamanji ([@kgamanji](https://github.com/kgamanji)), Apple
 
 ### Emeritus Leads
 
 - Gerred Dillon ([@gerred](https://github.com/gerred)), Mycelial
 - Paris Pittman ([@parispittman](https://github.com/parispittman)), Apple
 - Stephen Augustus ([@justaugustus](https://github.com/justaugustus)), Cisco
 
-## Working Groups
+## Working Groups And Teams
 
-### Governance
+### Governance WG
 
 [Charter](/governance/README.md)
 
 Facilitators:
 
 - Josh Berkus ([@jberkus](https://github.com/jberkus)), Red Hat
+- Dawn Foster ([@geekygirldawn](https://github.com/geekygirldawn)), VMware
 
-### Contributor Growth  
+### Contributor Growth WG
```

Comment:
As long as we are updating these, Paris is just doing maintainers circle now and hasn't been a facilitator in a while.